### ISSUE TITLE
Requiring aria-checked on an input[type=checkbox]

### ISSUE THREAD

**Issue by jongund** [validator/validator#1200]
URL being validated or code to reproduce error:
https://raw.githack.com/w3c/aria-practices/switch-checkbox/examples/switch/switch-checkbox.html

```
Element “input” is missing required attribute “aria-checked”.
```

When a checkbox is being used as widget the `.checked` property should be used instead of `arai-checked`.

**Comment by jongund** [comment]
Is already identified in Issue #1122.
https://github.com/validator/validator/issues/1122
